README.md

```diff
@@ -95,7 +95,7 @@ Unicode coverage makes Fira Code a great choice for mathematical writing:
 | **Android Studio** (2.3+, [instructions](https://github.com/tonsky/FiraCode/wiki/IntelliJ-products-instructions)) | **Adobe Dreamweaver** |
 | **Anjuta** (unless at the EOF) | **Delphi IDE** |
 | **AppCode** (2016.2+, [instructions](https://github.com/tonsky/FiraCode/wiki/IntelliJ-products-instructions)) | Standalone **Emacs** ([workaround](https://github.com/tonsky/FiraCode/wiki/Emacs-instructions)) |
-| **Atom** 1.1 or newer ([instructions](https://github.com/tonsky/FiraCode/wiki/Atom-instructions)) | **Godot** ([issue](https://github.com/godotengine/godot/issues/9961)) |
+| **Atom** 1.1 or newer ([instructions](https://github.com/tonsky/FiraCode/wiki/Atom-instructions)) |
```

Comment:
Applications in the right column need moving up 1 row each to fill the gap left behind

![Screenshot 2022-07-11 at 16 47 18](https://user-images.githubusercontent.com/3440094/178304176-3995d399-92ff-435c-9c2f-b3b912d440f2.png)

Reply:
Thanks, changed.